Makefile

```diff
@@ -212,7 +212,8 @@ deps: \
 
 .PHONY: goimports/install
 goimports/install:
-	GO111MODULE=off go get -u golang.org/x/tools/cmd/goimports
+	go get -u golang.org/x/tools/cmd/goimports
```

Comment:
if we delete `GO111MODULE=off `, `go.mod` of root directory may change. 🤔

Reply:
maybe it's okay, if developer already have goimports this section never run